src/vs/workbench/contrib/preferences/browser/settingsEditor2.ts

```diff
@@ -90,7 +90,7 @@ export class SettingsEditor2 extends EditorPane {
 	private static CONFIG_SCHEMA_UPDATE_DELAYER = 500;
 
 	private static readonly SUGGESTIONS: string[] = [
-		`@${MODIFIED_SETTING_TAG}`, '@tag:usesOnlineServices', '@tag:sync', `@${EXTENSION_SETTING_TAG}`
+		`@${MODIFIED_SETTING_TAG}`, '@tag:usesOnlineServices', '@tag:sync', `@${EXTENSION_SETTING_TAG}`, `@${FEATURE_SETTING_TAG}`
```

Comment:
Maybe the suggestions should include all options for `@feature:`, I'm not sure whether they should or not.

Reply:
added these and it seems to overwhelm the suggestions box. what do you think?